libs/json/src/test/test_json_simple.c

```diff
@@ -180,6 +181,23 @@ TEST_CASE(test_json_simple_decode){
     long long int intarr[8];
     int rc;
     int array_count;
+    
+    /*working with structures of arrrays. declearing variable*/
+    
+    struct test_structarr {
+        
+        bool bool_val2;
+        long long int int_val2;
+        char stringstruct2 [16];
+        
+    };
+    
+    struct test_structarr studarr[3];
+     struct test_jbuf tjb2; 
```

Comment:
sorry for those.
I just did another pull with the above corrections check on it.
thanks

On 5/23/16, Sterling Hughes <notifications@github.com> wrote:
>> @@ -180,6 +181,23 @@ TEST_CASE(test_json_simple_decode){
>>      long long int intarr[8];
>>      int rc;
>>      int array_count;
>> +
>> +    /*working with structures of arrrays. declearing variable*/
>> +
>> +    struct test_structarr {
>> +
>> +        bool bool_val2;
>> +        long long int int_val2;
>> +        char stringstruct2 [16];
>> +
>> +    };
>> +
>> +    struct test_structarr studarr[3];
>> +     struct test_jbuf tjb2;
>
> similar indent to the line before it (4 spaces) please
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/apache/incubator-mynewt-core/pull/58/files/4d363486b6ba9987692045c0563c7adbc6b5e84c#r64247285


-- 
"A Goal is a Dream with a Plan and a Dateline"
Do On to Others what you will like them to do on to you.'The Golden Rule'
Firefox Student Ambassadors Regional Lead Cameroon.
Software Developer,
Web Designer and Developer.
Music DJ . Artiste at Casky Black's Record